aiida/manage/configuration/options.py

```diff
@@ -102,6 +38,16 @@ def parse_option(option_name, option_value):
         'description': 'The timeout in seconds for calls to the circus client',
         'global_only': False,
     },
+    'db.batch_size': {
+        'key': 'db_batch_size',
+        'valid_type': 'int',
+        'valid_values': None,
+        'default': 100000,
+        'description':
+        'Batch size for bulk CREATE operations in the database. Avoids hitting MaxAllocSize of PostgreSQL'
+        '(1GB) when creating large numbers of database records in one go.',
+        'global_only': True,
```

Comment:
What is the reason to limit it to global option only? Maybe not the most common use case to have different batch sizes, but it is possible to have multiple profiles in one installation that connect to different databases on different machines with different memory limitations. Best to leave unconstrained I would say